src/org/omegat/gui/theme/DefaultFlatTheme.java

```diff
@@ -107,6 +108,13 @@ public static UIDefaults setDefaults(UIDefaults defaults, String lafId) {
         // #EEEEEE -> #575757; Darkest border
         Color statusAreaColor = adjustRGB(standardBgColor, 0x57 - 0xEE);
 
+        // load default colors
+        try {
+            loadDefaultColors(defaults);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
I know the codebase is not very consistent, but we should probably use `Log.log(e);`